scripts/start.sh

```diff
@@ -40,4 +40,4 @@ export APPGW_ENABLE_SAVE_CONFIG_TO_FILE="true"
 ./bin/appgw-ingress \
     --in-cluster=false \
     --kubeconfig="$KUBE_CONFIG_FILE" \
-    --verbosity=9
+    --verbosity=3
```

Comment:
why we need to change this ?

Reply:
For testing to better count the batches, will removed in the last final commit

Reply:
(opening to remember)